pkg/sql/schema_changer.go

```diff
@@ -1782,3 +1769,50 @@ func init() {
 	}
 	jobs.RegisterConstructor(jobspb.TypeSchemaChange, createResumerFn)
 }
+
+// queueCleanupJobs checks for columns and indexes that have been dropped
+// and queues a job for the garbage collector to delete the data.
+func (sc *SchemaChanger) queueCleanupJobs(
```

Comment:
I'd change this comment to something like "... checks if the completed schema change needs to start a child job to clean up dropped schema elements."